src/style/calculator.css

```diff
@@ -5,19 +5,26 @@
   justify-content: center;
   align-items: center;
   padding: 0 5% 0 0;
+  flex-direction: column;
+}
+
+.app h2 {
+  font-size: 2.5em;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  color: rgb(97, 95, 95);
 }
 
 .calculator {
-  width: 16em;
-  height: 18em;
-  border: 4px solid rgb(251, 251, 251);
```

Comment:
- [ ] Please make sure the heading and calculator are horizontally aligned (heading on the left side and calculator on the right side) to follow the wireframe design.

### Your design

![image](https://github.com/BiancaMe/math-magicians/assets/83803414/24e60940-bd42-42b3-b094-6089eda386a0)

### Wireframe design

![image](https://github.com/BiancaMe/math-magicians/assets/83803414/071c59cf-9e46-43b2-9080-30fa5c793bf2)

Reply:
Hello, @thecodechaser . You told me I must fix some eslint errors in my files without disabling linter errors. I'm still having problems with the Buttons.js file. 
Buttons.js : 
![image](https://github.com/BiancaMe/math-magicians/assets/37249375/4757c524-13a8-4f22-9940-9228ae1f1c1d)

The only way that I found to solve this is disabled the error. If you read this: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/3d1d26d89d492947cbf69f439deec9e7cfaf9867/docs/rules/no-static-element-interactions.md. 
You can see that : 
![image](https://github.com/BiancaMe/math-magicians/assets/37249375/3d03cea2-8024-4ffd-948b-f667ab1d4411)

I think that it is my case. What do you think? Do you have another solution?